README.rst

```diff
@@ -28,7 +28,8 @@ Install from IndexedConv folder:
 
 .. code-block:: bash
 
-    python setup.py install
+    git clone https://github.com/IndexedConv/IndexedConv.git
+    pip install IndexedConv
```

Comment:
To avoid confusion with the installation of the package on pypi, I would rather:
`cd IndexedConv`
`pip install .`